knexfile.js

```diff
@@ -0,0 +1,17 @@
+// Update with your config settings.
+
+module.exports = {
+  client: 'pg',
+  connection: {
+    host: '127.0.0.1',
```

Comment:
@pegaheinaliyan @bgard6977 For now, it would be fine.
Just comment: Usually, we use Environment Variables in order to hide the important parameters like the database host, ports, database name, etc. Also, in a production/staging, the database might be a different server from a web server.